vitrivr-engine-core/src/main/kotlin/org/vitrivr/engine/core/model/content/impl/cache/CachedImageContent.kt

```diff
@@ -16,6 +17,8 @@ import javax.imageio.ImageIO
  */
 class CachedImageContent(override val path: Path, image: BufferedImage) : ImageContent, CachedContent {
 
+    override val id: UUID = UUID.randomUUID()
```

Comment:
I would add this to the constructor. There might be use-cases, where one wants to define this explicitly.You can also use default values in the constructor.